### ISSUE TITLE
Configuration option to disable hook timeout.

### ISSUE THREAD

**Issue by mnaughto** [balderdashy/sails#3365]
For just running the application, it makes sense to die if hooks are taking a while to load, but I've found that when a debugger is attached, the code generally takes a lot longer to run, but isn't necessarily in an infinite loop or whatever. It'd be nice to be able to set an option based on environment to disable the timeout check so that I can run my debugger without any headaches. 

I suppose a workaround right now is to set the default timeout to something large.

We just upgraded to sails 11, and this safety check was not present in the previous version we were using (10.5?).

**Comment by sailsbot** [comment]
Thanks for posting, @mnaughto.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
